### PR TITLE
Add Polaris retextures and White Heavy Plate Carrier

### LINKED ISSUE #109: Placed eagles have BWMod items
Title.

Eagles placed down will have BWMod items in them (G36, Panzerfurst etc.).
It's a minor thing, but might be an idea to either change the transport items to something that better fits Theseus Services, or make them have nothing at all when placed at first in editor or with Zeus?

### PR DESCRIPTION
**Pull request by Kresky**
**When merged this pull request will:**
- Add Polaris DAGOR black retexture vehicle (Vanilla "black" is more midnight blue than black) - Preview: https://i.imgur.com/QRike3U.png
  - Also added custom tan variant (no vanilla tan) and green variant (using vanilla pacific NATO variant)
- Add `MACRO_SET_VEHICLE_CARGO` macro, which will clear the vehicle inventory of all items except for 4 First Aid Kits
  - Added the above macro to all current vehicles (including Eagle's, closes #109)
- Add white heavy plate carrier texture
  - Corrected some minor oversights from the heavy plate carriers added in 1.10.0 (eg. classnames in config.cpp)

### CODE REVIEW

File: addons/vehicles/CfgVehicles.hpp [hunk 1]
```diff
@@ -103,4 +103,118 @@ class CfgVehicles {
         crew = QCLASS(Unit_B_Contractor);
         typicalCargo[] = {QCLASS(Unit_B_Engineer)};
     };
+
+    #define MACRO_CLEAR_VEHICLE_CARGO \
```
Comment (Kresky) [inline addons/vehicles/CfgVehicles.hpp]: @jonpas Wasn't 100% on this one. Should we maybe add it as a macro to `main` instead, and use it on just about every vehicle, including optionals (BWA stuff etc.)?

Or would you rather we don't just do a macro to clear everything, but customize the inventory a bit more to fit Theseus Services (Though in our own units case, having the vehicle cargo be completely empty would be more convenient)

Comment (jonpas) [reply addons/vehicles/CfgVehicles.hpp]: I'd rather we don't clear everything, but modify accordingly. 

For Theseus itself, we would have to do it for every vehicle, including vanilla anyways.

File: addons/vests/CfgWeapons.hpp [hunk 2]
```diff
@@ -158,6 +158,13 @@ class CfgWeapons {
         picture = QPATHTOF(UI\vest_PlateCarrierHeavy_green_ca.paa);
         hiddenSelectionsTextures[] = {QPATHTOF(data\vest_PlateCarrierHeavy_green_co.paa)};
     };
+    class CLASS(Vest_PlateCarrierHeavy_White): V_PlateCarrierGL_rgr {
+        MACRO_PlateCarrierHeavy_COMMON
```
Comment (jonpas) [inline addons/vests/CfgWeapons.hpp]: This should be all uppercase.

File: addons/vehicles/CfgVehicles.hpp (hunk 1 above)
Comment (Kresky) [reply addons/vehicles/CfgVehicles.hpp]: Maybe just add only 4 First Aid Kits (Similar to the Offroad currently)?

Comment (jonpas) [reply addons/vehicles/CfgVehicles.hpp]: That sounds good.

File: addons/vests/CfgWeapons.hpp (hunk 2 above)
Comment (Kresky) [reply addons/vests/CfgWeapons.hpp]: Changed to all uppercase

File: addons/vehicles/stringtable.xml [hunk 3]
```diff
@@ -19,5 +19,32 @@
             <Japanese>武装オフロード (黒)</Japanese>
             <French>Tout-terrain armé (Noir)</French>
         </Key>
+        <Key ID="STR_TACS_Vehicles_Polaris_Black">
+            <English>Polaris DAGOR (Black)</English>
+        </Key>
+        <Key ID="STR_TACS_Vehicles_Polaris_Armed_Black">
+            <English>Polaris DAGOR Armed (Black)</English>
```
Comment (jonpas) [inline addons/vehicles/stringtable.xml]: ACE3 strings [specify the weapon](https://github.com/acemod/ACE3/blob/d684a8042e56810fc916d9d205f228cb7de713ed/addons/realisticnames/stringtable.xml#L3210).
We should follow that same as for [BWA3 optional](https://github.com/Theseus-Aegis/TheseusServices/blob/master/optionals/variants_bwa3/stringtable.xml#L14).

File: optionals/variants_melb/CfgVehicles.hpp [hunk 4]
```diff
@@ -8,6 +8,7 @@ class CfgVehicles {
 
     // Theseus Black
     class CLASS(MELB_I_AH6M_H_Theseus_Black): MELB_AH6M_H {
+        MACRO_SET_VEHICLE_CARGO
```
Comment (jonpas) [inline optionals/variants_melb/CfgVehicles.hpp]: What do MELBs have inside by default that doesn't fit a PMC?

File: addons/vehicles/CfgVehicles.hpp [hunk 5]
```diff
@@ -1,6 +1,7 @@
 class CfgVehicles {
     class B_Heli_Light_01_F;
     class CLASS(Heli_I_MH6_Blue): B_Heli_Light_01_F {
+        MACRO_SET_VEHICLE_CARGO
```
Comment (jonpas) [inline addons/vehicles/CfgVehicles.hpp]: What does this contain that doesn't fit a PMC?

File: addons/vehicles/CfgVehicles.hpp [hunk 6]
```diff
@@ -27,6 +28,7 @@ class CfgVehicles {
         class EventHandlers;
     };
     class CLASS(Offroad_I_Black): Offroad_01_unarmed_base_F {
+        MACRO_SET_VEHICLE_CARGO
```
Comment (jonpas) [inline addons/vehicles/CfgVehicles.hpp]: What does this contain that doesn't fit a PMC?

Comment (Kresky) [reply addons/vehicles/CfgVehicles.hpp]: Misremembered on my part, the offroad actually already comes with 4 First Aid Kits, so I'll remove the macro here.

File: optionals/variants_melb/CfgVehicles.hpp (hunk 4 above)
Comment (Kresky) [reply optionals/variants_melb/CfgVehicles.hpp]: Some reason thought MELBs came with more stuff when I added the macro. Checked now, it comes with 4 first aid kits already, as well as a GPS and toolkit. I'll remove the macro.

File: addons/vehicles/CfgVehicles.hpp (hunk 5 above)
Comment (Kresky) [reply addons/vehicles/CfgVehicles.hpp]: It comes with some smokes, MXC rifles and ammo and steerable parachutes. So not much "issue" at all compared to something like the Eagles and the german equipment they add.

File: addons/vehicles/stringtable.xml (hunk 3 above)
Comment (Kresky) [reply addons/vehicles/stringtable.xml]: Names corrected

File: addons/main/script_macros.hpp [hunk 7]
```diff
@@ -60,3 +60,9 @@
         backpack = #BACKPACK; \
         count = COUNT; \
     }
+
+#define MACRO_SET_VEHICLE_CARGO \
+        class TransportBackpacks {}; \
+        class TransportItems {MACRO_ADDITEM(FirstAidKit,4)}; \
+        class TransportMagazines {}; \
+        class TransportWeapons {};
```
Comment (jonpas) [inline addons/main/script_macros.hpp]: I would try to make it similar to other vanilla vehicles, so some weapon (from Theseus Services ideally), a GPS and whatever else is there.

Rename macro to indicate land vehicle, as air would have parachutes in addition.

File: addons/main/script_macros.hpp [hunk 8]
```diff
@@ -61,8 +61,30 @@
         count = COUNT; \
     }
 
-#define MACRO_SET_VEHICLE_CARGO \
-        class TransportBackpacks {}; \
-        class TransportItems {MACRO_ADDITEM(FirstAidKit,4)}; \
-        class TransportMagazines {}; \
-        class TransportWeapons {};
+#define MACRO_SET_LANDVEHICLE_CARGO \
```
Comment (jonpas) [inline addons/main/script_macros.hpp]: No "SET".

Comment (Kresky) [reply addons/main/script_macros.hpp]: Removed set from air and land macros

**Pull request merged**